components/collection/CollectionCard.vue

```diff
@@ -11,10 +13,16 @@
         :name="collection.name || ''"
         :image="image" />
     </nuxt-link>
+
+    <template v-else>
+      <NeoSkeleton :no-margin="true" :rounded="false" height="112px" />
+      <CollectionDetail :is-loading="true" :nfts="[]" name="" />
+    </template>
```

Comment:
would it be possible to keep this outside `CollectionCard` component? maybe just loop through skeleton in `CollectiionList` component (or create `NeoCollectionCardSkeleton`)

Reply:
If we create another component for the skeleton, the CSS for this card component could not be reused.  What's more, if we want to make some changes to one component, we should also take care of the other related component.

Reply:
Putting it in one file seems to make it easier to synchronize updates to the skeleton when the html structure changes?